### ISSUE TITLE
Arguments of scrollMouse() function in Windows are reversed?

### ISSUE THREAD

**Issue by tkomde** [hurdlegroup/robotjs#20]
I appreciate your work on robotjs!

It seems that the arguments of scrollMouse() function in Windows are reversed.

## Expected Behavior

In MacOS, following code execute a down scroll. This is in accordance with the specifications.

```
const { scrollMouse } = require("@hurdlegroup/robotjs");
scrollMouse(0, -20);
```

## Current Behavior

In Windows, the code execute a *left scroll*.
Following code execute a down scroll.

```
const { scrollMouse } = require("@hurdlegroup/robotjs");
scrollMouse(20, 0);
```

## Possible Solution

- Flip arguments order in Windows.
- Also, a direction may be reversed to match the MacOS. 

## Your Environment

* RobotJS version: 0.11.5
* Node.js version: 22.0.0
* npm version: 10.5.1
* Operating System: Windows11 Home (win32 / x64)

**Comment by DanielMcAssey** [comment]
Thanks for reporting this, this should now be resolved in v0.11.7

**Comment by tkomde** [comment]
@DanielMcAssey 

Thank you for quick fix. I confirmed that the issue is solved!
